### DISCUSSION TITLE
Confused about differential diffusion script

### DISCUSSION THREAD

**Discussion by 23pennies** [vladmandic/sdnext#2963] [Q&A]
https://github.com/vladmandic/automatic/blob/c1dfb1b28e37f286107b44740ccf6d1733233992/scripts/differential_diffusion.py#L1900C9-L1900C18
In this line, the input parameter image_map, which I understand as the user-defined mask, is discarded and set to None. With model also being None, the script just errors out with "no image map". Am I misunderstanding something?

**Comment by vladmandic** [comment]
i think you're right and that's a leftover from early prototype.
since you've identified it fully, why not submit a pr directly?
if not, i'll take care of it.
